omegaconf/config.py

```diff
@@ -399,32 +399,32 @@ def merge_with(self, *others):
         Config._re_parent(self)
 
     @staticmethod
-    def _resolve_value(root_node, inter_type, inter_key):
+    def _resolve_value(root_node, inter_type, *args):
```

Comment:
I prefer that we keep the value as a string until we actually call the function.